modules/statistics/src/main/resources/SLING-INF/content/Extensions/Views/UserStatistics.json

```
@@ -0,0 +1,7 @@
+{
+    "jcr:primaryType": "cards:Extension",
+    "cards:extensionPointId": "cards/coreUI/view",
+    "cards:extensionName": "User Statistics",
```

I think it's better like this:

```suggestion
    "cards:extensionName": "Statistics",
```

(this is visible in the browser window/tab title)